schemas/service.proto

```diff
@@ -33,6 +33,17 @@ message PutWfSpecRequest {
   string entrypoint_thread_name = 6;
   optional WorkflowRetentionPolicy retention_policy = 8;
   reserved 9;
+  optional AllowedUpdateType allowed_updates = 10;
```

Comment:
Please make this not an optional field. It will default to the enum value zero. That is backwards compatible.